app/components/post_list/post_list.android.js

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {StyleSheet} from 'react-native';
+import RecyclerViewList, {DataSource, RecyclerRefreshControl} from 'react-native-recyclerview-list';
+
+import {ListTypes} from 'app/constants';
+
+import PostListBase from './post_list_base';
+
+const SCROLL_UP_MULTIPLIER = 3.5;
+
+export default class PostList extends PostListBase {
+    constructor(props) {
+        super(props);
+
+        this.contentOffsetY = 0;
```

Comment:
Why store `contentOffsetY` and `contentHeight` on the PostList? I only see them being used in `handleScroll` after they've been set from the event

Reply:
we need to initialize the value as 0 cause the first time it is run it has to be set instead of being undefined

Reply:
Yeah, but the code where we're using them looks to me like
```
handleScroll = (contentHeight) => {
     this.contentHeight = contentHeight;
     if (this.contentHeight > ...) {...}
}
```
so I don't think they need to be member variable since that's basically equivalent to
```
handleScroll = (contentHeight) => {
     if (contentHeight > ...) {...}
}
```